### PR TITLE
Done

### PR DESCRIPTION
**Pull request by thomashines**

### CODE REVIEW

**Comment by bhollan** [comment]
Hi @thomashines !

Looking at your changes, almost everything looks good.  Correct me if I'm wrong, but I don't see enough closing `</div>` tags in your changes to `index.html`.  If you can add in that change, we can merge your profile in.

As always, feel free to Ask A Question if you need any help getting things squared away.
CC: @aturkewi

**Comment by thomashines** [comment]
Brian, apologies, I have no idea what you’re talking about? Maybe you have the wrong guy? 

Tom

**Comment by bhollan** [comment]
No, @thomashines , I'm quite sure you're it.

But if you don't remember doing the `deploy-on-day-one` lesson through Learn with the Flatironschool, then it's not important.

If I don't hear back from you, this pull request will be closed.

**Comment by thomashines** [comment]
I'm sorry, maybe I accidentally triggered a pull request? This was a long time ago, right? There have been many stray HTML tags since! If there's anything you need me to review or correct, let me know. If I requested input, it may have been by accident. If so, apologies.

Tom

File: index.html [hunk 1]
```diff
@@ -24,7 +24,18 @@
           <h2>We're a group of people on a journey to learn to love code.</h2>
         </div>
 
-
+        <div class="roster-cards-container">
```
Comment (aturkewi) [inline index.html]: Delete this added line

File: index.html [hunk 2]
```diff
@@ -24,7 +24,18 @@
...
+                <h3 class="view-profile-text">View Profile</h3>
+              </div>
+              <div class="card-text-container">
+                <h4 class="student-name">Thomas H</h4>
+                <p class="student-location">New York, NY</p>
+              </div>
+            </a>
+          </div>
 
         <div class="roster-cards-container">
```
Comment (aturkewi) [inline index.html]: Move all your changes _below_ this line

**Comment by aturkewi** [comment]
Hello! We recently made a change to this repo that caused all open PRs to be close and we are not able to re-open them. I'm very sorry about the inconvenience, but I just wanted to let you know why this PR was close and that we still welcome your profile on the site! So if you're still interested in having your profile up, please feel free to open another PR.

**Pull request closed without merging**